From 894ed10f90343e2e82c21b31d8e5de7305b398dd Mon Sep 17 00:00:00 2001
From: fupan li <fupan.li@windriver.com>
Date: Thu, 7 Nov 2013 17:10:23 +0800
Subject: [PATCH 1/2] clk: add the DW DMA clk

BayTrail has two DMA engines, which can be enumerated by PCI and ACPI.
But the driver only support the PCI enumerated now, and its dev id is expressed
as it's pci id, instead of the dw_dmac0, dw_dmac1, correct it when registering its clk.

The YP BSP missed to register its another clk, here added it.

Signed-off-by: fupan li <fupan.li@windriver.com>
---
 arch/x86/platform/byt/byt-board.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/x86/platform/byt/byt-board.c b/arch/x86/platform/byt/byt-board.c
index 47c8bbe..b2f1caf 100644
--- a/arch/x86/platform/byt/byt-board.c
+++ b/arch/x86/platform/byt/byt-board.c
@@ -55,7 +55,10 @@ static int byt_clk_setup()
 		return PTR_ERR(clk);
 
 	clk_register_clkdev(clk, NULL, "0000:00:1e.5");
-	clk_register_clkdev(clk, "hclk", "dw_dmac.0");
+
+	/*register the clock for DW DMA engines*/
+	clk_register_clkdev(clk, "hclk", "0000:00:18.0");
+	clk_register_clkdev(clk, "hclk", "0000:00:1e.0"); 
 
 	return 0;
 }
-- 
1.8.4.93.g57e4c17

